AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -100,7 +106,8 @@ private extension SessionHeaderView {
     
     var actionsMenuMobile: some View {
         Menu {
-            actionsMenuMobileStopButton
+            session.isActive ? actionsMenuStopButton : nil
+            (session.isDormant || session.isFixed) ? actionsMenuDeleteButton : nil
```

Comment:
let's move this to extension for session so this looks like:
```Swift
session.deletable ? actionsMenuDeleteButton : nil
```

this way we can use it in other parts of the app and change it in a single place